napari/_qt/qt_main_window.py

```diff
@@ -12,17 +12,17 @@
 app.use_app(API_NAME)
 del app
 
-from qtpy.QtWidgets import (
+from qtpy.QtWidgets import (  # noqa
```

Comment:
why do we end up with a `# noqa` here?

Reply:
that code breaks  [Module level import not at top of file (E402) ](https://lintlyci.github.io/Flake8Rules/rules/E402.html) ... but I didn't want to just move the imports, since that can sometimes alter behavior.  so rather than globally ignoring `E402`, I copped out with the  `# noqa`.  see my comment below about `#noqa` ... but I can also take a closer look to make sure moving the imports wouldn't break anything here.

Reply:
those imports might need to be there, I can't fully remember, but I do think there were a couple places like that due to qt stuff